dotCMS/src/main/java/com/dotmarketing/quartz/QuartzUtils.java

```diff
@@ -40,29 +37,7 @@
 	
 	private static Map<String, TaskRuntimeValues> runtimeTaskValues = new HashMap<String, TaskRuntimeValues>();
 	
-	/**
-	 * 
-	 * Lists all jobs scheduled through the sequential scheduler, the sequential scheduler
-	 * let you run only one job at a time
-	 * 
-	 * @return
-	 */
-	public static List<ScheduledTask> getSequentialScheduledTasks() throws SchedulerException {
-		Scheduler sched = DotSchedulerFactory.getInstance().getSequentialScheduler();
-		return getScheduledTasks(sched, true, null);
-	}
-	
-	/**
-	 * 
-	 * Lists all jobs scheduled through the standard scheduler, the standard scheduler
-	 * let you run multiple jobs in parallel
-	 * 
-	 * @return
-	 */
-	public static List<ScheduledTask> getStandardScheduledTasks() throws SchedulerException {

```

Comment:
I get a `java.lang.NoSuchMethodError: com.dotmarketing.quartz.QuartzUtils.getStandardScheduledTasks` from site search on this branch